package.json

```diff
@@ -10,7 +10,7 @@
     "@typescript-eslint/parser": "^5.12.0",
     "eslint": "^8.9.0",
     "log-update": "^5.0.0",
-    "parcel": "^2.3.2",
+    "parcel": "^2.10.2",
```

Comment:
Updated `parcel` to fix the errors when trying to package things with Node.js 18

See https://github.com/parcel-bundler/parcel/issues/8005

Before this change:
```sh
$ node .config/do.mjs package
Bundling Core tests with Parcel...
@parcel/optimizer-terser: You must provide the URL of lib/mappings.wasm by
calling SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) before using
SourceMapConsumer
  Error: You must provide the URL of lib/mappings.wasm by calling
  SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) before using
  SourceMapConsumer
  at readWasm
  (/home/eric/Documents/github/vscode-rewrap/node_modules/terser/node_modules/source-map/lib/read-wasm.js:8:13)
  at wasm
  (/home/eric/Documents/github/vscode-rewrap/node_modules/terser/node_modules/source-map/lib/wasm.js:25:16)
  at
  /home/eric/Documents/github/vscode-rewrap/node_modules/terser/node_modules/source-map/lib/source-map-consumer.js:264:14
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async SourceMap
  (/home/eric/Documents/github/vscode-rewrap/node_modules/terser/dist/bundle.min.js:19362:20)
  at async minify
  (/home/eric/Documents/github/vscode-rewrap/node_modules/terser/dist/bundle.min.js:27714:41)
  at async Object.optimize
  (/home/eric/Documents/github/vscode-rewrap/node_modules/@parcel/optimizer-terser/lib/TerserOptimizer.js:131:16)
  at async PackagerRunner.optimize
  (/home/eric/Documents/github/vscode-rewrap/node_modules/@parcel/core/lib/PackagerRunner.js:364:20)
  at async PackagerRunner.getBundleResult
  (/home/eric/Documents/github/vscode-rewrap/node_modules/@parcel/core/lib/PackagerRunner.js:255:15)
  at async PackagerRunner.getBundleInfo
  (/home/eric/Documents/github/vscode-rewrap/node_modules/@parcel/core/lib/PackagerRunner.js:239:9)
Error running: npx --silent parcel build core/test --cache-dir .obj/parcel
@parcel/optimizer-terser: You must provide the URL of lib/mappings.wasm by
calling SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) before using
SourceMapConsumer
  Error: You must provide the URL of lib/mappings.wasm by calling
  SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) before using
  SourceMapConsumer
  at readWasm
  (/home/eric/Documents/github/vscode-rewrap/node_modules/terser/node_modules/source-map/lib/read-wasm.js:8:13)
  at wasm
  (/home/eric/Documents/github/vscode-rewrap/node_modules/terser/node_modules/source-map/lib/wasm.js:25:16)
  at
  /home/eric/Documents/github/vscode-rewrap/node_modules/terser/node_modules/source-map/lib/source-map-consumer.js:264:14
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async SourceMap
  (/home/eric/Documents/github/vscode-rewrap/node_modules/terser/dist/bundle.min.js:19362:20)
  at async minify
  (/home/eric/Documents/github/vscode-rewrap/node_modules/terser/dist/bundle.min.js:27714:41)
  at async Object.optimize
  (/home/eric/Documents/github/vscode-rewrap/node_modules/@parcel/optimizer-terser/lib/TerserOptimizer.js:131:16)
  at async PackagerRunner.optimize
  (/home/eric/Documents/github/vscode-rewrap/node_modules/@parcel/core/lib/PackagerRunner.js:364:20)
  at async PackagerRunner.getBundleResult
  (/home/eric/Documents/github/vscode-rewrap/node_modules/@parcel/core/lib/PackagerRunner.js:255:15)
  at async PackagerRunner.getBundleInfo
  (/home/eric/Documents/github/vscode-rewrap/node_modules/@parcel/core/lib/PackagerRunner.js:239:9)

Packaging & Optimizing...
🚨 Build failed.
```